test/fixtures/dialects/spark3/create_database.sql

```diff
@@ -20,3 +20,7 @@ CREATE DATABASE IF NOT EXISTS customer_db;
 CREATE DATABASE IF NOT EXISTS customer_db
 COMMENT 'This is customer database' LOCATION '/user'
 WITH DBPROPERTIES ("ID" = "001", "Name" = 'John');
+
+-- Create `inventory_db` Database
+CREATE DATABASE inventory_db
+COMMENT 'This database is used to maintain Inventory';
```

Comment:
In the future best to add these extra test cases in a separate PR. But thanks for adding them 😄